addons/lunch/static/src/xml/lunch.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <t t-name="lunch.PreviousOrder">
```

Comment:
The prefix is not necessary, as we are in the lunch module. However you should check your template names 
- we don't use camel case
- should probably be lunch_order_widget_previous_order or something like that.